makefile.shade

```diff
@@ -153,16 +153,16 @@ var RC_FILES = '${FindAllFiles("Resource.rc", BOOTSTRAPPER_EXE_NAME, BOOTSTRAPPE
             var environmentRuntimeTargetOS = Environment.GetEnvironmentVariable("RUNTIME_TARGET_OS");
             var configRuntimeTargetOS = "";
         
-            Exec(MSBUILD, bootstrapperProj + " /p:Configuration=" + Configuration2 + ";Platform=Win32;TargetFramework=aspnet50");
```

Comment:
These changes will break the C++ build. We need to update dnx.vcxproj. While we're at it, lets just call it what it really is:

- coreclr
- clr

We should change the property to `Runtime=CLR` and `Runtime=CoreCLR`